util/test/demos/gl/gl_queries_in_use.cpp

```diff
@@ -65,7 +65,7 @@ RD_TEST(GL_Queries_In_Use, OpenGLGraphicsTest)
     glAttachShader(program, vs);
     glAttachShader(program, fs);
 
-    GLchar *posOnly = "gl_Position";
+    GLchar *posOnly = (GLchar *)"gl_Position";
```

Comment:
I think actually making `posOnly` a `const GLchar *` would work, that's what `glTransformFeedbackVaryings` takes and I'm not sure why I didn't in the first place.